docs/data-types/hyperloglogs.md

```diff
@@ -6,10 +6,48 @@ description: >
     Introduction to the Redis HyperLogLog data type
 ---
 
-HyperLogLog is a data structure that estimates the cardinality of a set. As a probabilistic data structure, HyperLogLog trades perfect accuracy for efficient space utilization.
+HyperLogLog is a probabilistic data structure that estimates the cardinality of a set. As a probabilistic data structure, HyperLogLog trades perfect accuracy for efficient space utilization.
 
 The Redis HyperLogLog implementation uses up to 12 KB and provides a standard error of 0.81%.
 
+Counting unique items usually requires an amount of memory
+proportional to the number of items you want to count, because you need
+to remember the elements you have already seen in the past in order to avoid
+counting them multiple times. However there is a set of algorithms that trade
+memory for precision: you end with an estimated measure with a standard error,
```

Comment:
```suggestion
However, a set of algorithms exist that trade memory for precision: they yield an estimated measure with a standard error, which, in the case of the Redis implementation, is less than 1%.
```

This improves the readability here in my opinion

Reply:
I think:

```
However, a set of algorithms exist that trade memory for precision: they return an estimated measure with a standard error, which, in the case of the Redis implementation for HyperLogLog, is less than 1%.
```

Reply:
Also, you'd have to remove "However there is a set of algorithms that trade" from line number 16.

Reply:
So sorry Accidentally missed involving the lines accurately

Reply:
Should I create a fresh suggestion for this ? @elena-kolevska 
"However, a set of algorithms exist that trade memory for precision: they return an estimated measure with a standard error, which, in the case of the Redis implementation for HyperLogLog, is less than 1%."

Reply:
Can I open a PR to your branch ? @elena-kolevska